var/spack/repos/builtin/packages/stat/package.py

```diff
@@ -59,20 +61,18 @@ class Stat(Package):
 
     patch('configure_mpicxx.patch', when='@2.1.0')
 
-    def install(self, spec, prefix):
-        configure_args = [
-            "--enable-gui",
-            "--prefix=%s" % prefix,
+    def configure_args(self):
+        spec = self.spec
+        args = [
             "--with-launchmon=%s"   % spec['launchmon'].prefix,
             "--with-mrnet=%s"       % spec['mrnet'].prefix,
             "--with-graphlib=%s"    % spec['graphlib'].prefix,
             "--with-stackwalker=%s" % spec['dyninst'].prefix,
-            "--with-libdwarf=%s"    % spec['libdwarf'].prefix
+            "--with-libdwarf=%s"    % spec['libdwarf'].prefix,
+            "--with-python=%s/bin/python" % spec['python'].prefix
```

Comment:
STAT is incompatible with Python 3 anyway. I should explicitly call this out... will work on that.